types/filters/parse.go

```diff
@@ -221,6 +221,24 @@ func (filters Args) ExactMatch(field, source string) bool {
 	return false
 }
 
+// OnlyOneExactMatch returns true if there is only one filters and the source matches exactly this one.
+func (filters Args) OnlyOneExactMatch(field, source string) bool {
+	fieldValues, ok := filters.fields[field]
+	//do not filter if there is no filter set or cannot determine filter
+	if !ok || len(fieldValues) == 0 {
+		return true
```

Comment:
@thaJeztah yeah I was going to do it too but that's how `ExactMatch` is implemented (it there is no filter, `ExactMatch` on something will return true, which is a little bit misleading).